### DISCUSSION TITLE
Auto complete is not working with python

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [helix-editor/helix#7809] [Troubleshooting]
This is the output of --health check 
![image](https://github.com/helix-editor/helix/assets/72500236/572364e8-6f0e-46d8-a5c4-99bc7f3c75ef)
- my helix version is helix 23.05 (7f5940be)
- I get the message when openining a new python file 
![image](https://github.com/helix-editor/helix/assets/72500236/31a72de5-37cf-420a-801c-e98e41642652)
Language server exited

**Comment by the-mikedavis** [comment]
Try running in verbose mode and check the log file: https://github.com/helix-editor/helix/wiki/FAQ#access-the-log-file

The log will show the communication between Helix and the language server when running in verbose mode (`hx -v`).

**Reply by abdelkareemkobo** [reply]
This what i get 
2023-09-14T10:45:41.059 helix_vcs [INFO] failed to obtain current head name for /home/kareem/Desktop/nlp_projects/qdrant_learning/simple_neural_search.py
2023-09-14T10:45:41.059 helix_view::editor [ERROR] Failed to initialize the LSP for `source.python` { cannot find binary path }
